apollo-backend/src/main/java/io/logz/apollo/websockets/WebSocketServer.java

```diff
@@ -62,6 +67,11 @@ private Server createWebsocketServer(ApolloConfiguration configuration, Authenti
     @PostConstruct
     public void start() {
         try {
+            if (slaveService.isRunningInSlaveMode()) {
```

Comment:
Remove

Reply:
Can't remove:
Caused by: java.lang.IllegalArgumentException: Can't change the args! Different args were already assigned!
OLD: [secret=secret, on.address=0.0.0.0, on.port=64102]
NEW: [secret=secret, on.address=0.0.0.0, on.port=64118]